apps/widgets/src/pages/admin-dashboard.tsx

```diff
@@ -0,0 +1,57 @@
+import { useEffect, useState } from 'react';
+import type { Widget, WidgetStatus } from '@walless/graphql';
+import { qlClient } from '@walless/graphql';
```

Comment:
please pull the latest code, `qlClient` no longer available on `@walless/graphql`.